build.sbt

```diff
@@ -1,27 +1,25 @@
-import com.typesafe.sbt.SbtScalariform._
 
 import com.typesafe.sbt.SbtMultiJvm
-
 import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
 
+organization := "com.sclasen"
 name := "akka-zk-cluster-seed"
-
 version := "0.1.9-SNAPSHOT"
 
 scalaVersion := "2.11.8"
-
-crossScalaVersions := Seq("2.11.8")
+crossScalaVersions := Seq(scalaVersion.value, "2.12.1")
 
 val akkaVersion = "2.4.12"
+val akkaHttpVersion = "10.0.1"
 
 val akkaDependencies = Seq(
   "com.typesafe.akka" %% "akka-actor" % akkaVersion,
   "com.typesafe.akka" %% "akka-cluster" % akkaVersion,
   "com.typesafe.akka" %% "akka-slf4j" % akkaVersion,
+  "com.typesafe.akka" %% "akka-http-core" % akkaHttpVersion,
   "org.slf4j" % "log4j-over-slf4j" % "1.7.7",
   "ch.qos.logback" % "logback-classic" % "1.1.2",
-  "io.spray" %% "spray-json" % "1.2.6",
-  "io.spray" %% "spray-client" % "1.3.2"
+  "io.spray" %% "spray-json" % "1.3.2"
 ).map(_ % Provided)
```

Comment:
I'm not sure if that should be provided, as I understand not everyone needs Exhibitor, but maybe it would be better idea, to move ExhibitorClient with its dependencies into different artifact? WDYT?